quiz.rb

```diff
@@ -6,8 +6,11 @@
 # it should accept strings or numbers in its arguments
 # it should return false if age is less than minimum_age
 
+
+
 def is_of_age? age, minimum_age
-  # your code goes here
+ if age < minimum age
```

Comment:
Since you're using this method in later methods, you want it to return a value instead of printing a value to the console. The logic is right tho